docs/examples/roles/install.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Install sensu-backend binary
+  hosts: backends
+  roles:
+     - sensu.sensu_go.install
+  vars:
+    components: [ sensu-go-cli ]
+
+- name: Install sensu-agent binary from testing repository
+  hosts: agents
+  roles:
+     - sensu.sensu_go.install
+  vars:
+    components: [ sensu-go-agent ]
+    channel: testing
```

Comment:
Would you mind adding a task that installs `sensuctl` to localhost? Otherwise I think the fact that the role allows us to install `sensuctl` kind of gets lost.